DROD/CharacterDialogWidget.cpp

```diff
@@ -4577,6 +4596,11 @@ void CCharacterDialogWidget::PopulateCommandListBox()
 	this->pActionListBox->AddItem(CCharacterCommand::CC_WaitForRemains, g_pTheDB->GetMessageText(MID_WaitForRemains));
 	this->pActionListBox->AddItem(CCharacterCommand::CC_WaitForWeapon, g_pTheDB->GetMessageText(MID_WaitForWeapon));
 
+	this->pActionListBox->AddItem(CCharacterCommand::CC_LogicalWaitAnd, L"Logic Wait And");
```

Comment:
I'm brainstorming whether alternative command names might be more descriptive to the lay user.  For instance:

"Wait for All"
"Wait for Any"
"Wait for Exactly One"
"Wait for Conditions End"

or

"Wait for All of"
"Wait for Any of"
"Wait for Exactly One of"
"Wait for Expressions End"

etc.

Just a thought.

Reply:
Yeah, I'm thinking the UX on these names will probably be better, especially for users who don't have familarity with formal logic terms. The "of" versions provide more clarity in text, but might add a little extra clutter visual when editing as the script editor will be do some stuff to make these commands noticable.

Reply:
Yeah, I'm thinking the UX on these names will probably be better, especially for users who don't have familarity with formal logic terms. The "of" versions provide more clarity in text, but might add a little extra clutter visual when editing as the script editor will be do some stuff to make these commands noticable.